website/public/js/controllers/notificationCtrl.js

```diff
@@ -53,6 +60,23 @@ habitrpg.controller('NotificationCtrl',
        Notification.mp(mana);
     });
 
+    $rootScope.$watch('user.stats.lvl', function(after, before) {
+      if (after <= before) return; 
+      Notification.lvl();
+      $rootScope.playSound('Level_Up');
+      if (User.user._tmp && User.user._tmp.drop && (User.user._tmp.drop.type === 'Quest')) return;
+      if (after === 3) return; // Drop system unlock. FIXME can we do this without hardcoding?
```

Comment:
For now it should be in the controller. If there's a need to have it multiple places, we can put it in a milestones service.